### ISSUE TITLE
Adding MGLMapView as a subview breaks layout

### ISSUE THREAD

**Issue by osca** [NAXAM/mapbox-ios-binding#8]
When I add the mapbox mapview (MGLMapView) as a subview, my layout breaks somehow.

`
MGLMapView routeMapView = new MGLMapView(RouteView.Frame);
RouteView.AddSubview(routeMapView);
`

Without mapview in the bottom-left corner:
![img_4356](https://user-images.githubusercontent.com/148895/38514128-965d3e54-3c30-11e8-8393-dc9d899364a9.PNG)

With mapview put in the frame of the bottom-left corner:
![img_4355](https://user-images.githubusercontent.com/148895/38514154-a4b7b592-3c30-11e8-9506-5ad0008a63f6.PNG)

Can somebody please explain this to me? I have two other views, where the mapview is working pretty well. But there the mapview is starting in the superview of the view controller at point 0,0.

I almost tried everything to get it running. Also added it with the XIB editor with the same result.

Thanks for any help or advice.

**Comment by fleuverouge** [comment]
Hi @osca I've tried to put the map view inside a subview and had no problem.

<img width="375" alt="mapbox" src="https://user-images.githubusercontent.com/10191185/38660939-524b8cc2-3e59-11e8-9c1c-8ed7bc13d766.png">

 Please check your constraints / frames thoroughly.

**Comment by osca** [comment]
Thank you for your answer and I think you are right checking my constraints. I saw that i had translatesAutoresizingMaskIntoConstraints="NO" in my xib in all subviews and this broke the layout somehow. Changing it to YES made it. I tried it before with creating constraints manually, but it didn't help. Maybe it is a good hint for other people having this problem. This can be closed now and I thank you for your help.
